src/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -1576,7 +1576,7 @@ public bool KeepAlive
         {
             get
             {
-                return false;
```

Comment:
It is un-implemented in CoreFx.  

This is the comment in the code:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Requests/src/System/Net/FtpWebRequest.cs#L1585

> // We don't support connection pooling, so just silently ignore this.